sp800-63c/sec4_federation.md

```diff
@@ -30,7 +30,7 @@ IdPs and RPs MAY act as their own authorities of who to federate with or MAY ext
 
 Protocols requiring the transfer of keying information SHALL use a secure method to establish such keying information needed to operate the federated relationship during the registration process, including any shared secrets or public keys. Any symmetric keys used in this relationship SHALL be unique to a pair of federation participants.
 
-Federation relationships SHALL establish parameters regarding expected and acceptable identity assurance level (IAL) and authentication assurance level (AAL) in connection with the federated relationship. For example, a given RP may only accept IAL 2 identity proofing from a given IdP that has been vetted at that level.
+Federation relationships SHALL establish parameters regarding expected and acceptable identity assurance level (IAL) and authentication assurance level (AAL) in connection with the federated relationship.
```

Comment:
If I remember correctly, we relaxed requirements for federations to accredit particular AAL/IAL levels, but the context here is manual registration -- they do need to agree in the parameters of their federation.